src/common/interop/PowerToys.Interop.vcxproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <AssemblyTitle>PowerToys.Interop</AssemblyTitle>
     <AssemblyCompany>Microsoft Corp.</AssemblyCompany>
-    <AssemblyCopyright>Copyright (C) 2022 Microsoft Corp.</AssemblyCopyright>
+    <AssemblyCopyright>Copyright (C) 2023 Microsoft Corp.</AssemblyCopyright>
```

Comment:
Wondering if these lines are needed? Shouldn't it be handled by `Directory.Build.props` file?

Reply:
You're right. Let me test and I'll update.

Reply:
Looks like it was unneeded. Thanks for spotting it.